apps/picsa-server/supabase/types/index.ts

```diff
@@ -264,6 +264,36 @@ export interface Database {
         }
         Relationships: []
       }
+      translations: {
+        Row: {
+          created_at: string
+          en: string | null
+          id: number
+          mw_ny: string | null
+          sw: string | null
+          tg: string | null
+          zm_ny: string | null
+        }
```

Comment:
(+) - Thanks for remembering to export the definitions, makes it really nice to be able to write type-safe code in the services and display component